projects/plugins/jetpack/_inc/client/components/jetpack-benefits/index.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import { DisconnectCard } from '@automattic/jetpack-connection';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { getRedirectUrl } from '@automattic/jetpack-components';
+import './style.scss';
+
+/**
+ * Shows a list of benefits that Jetpack provides.
+ *
+ * @param {object} props - The component props.
+ * @param {Array} props.siteBenefits - An array of site benefits.
+ * @returns {React.Component} - The JetpackBenefits component.
+ */
+const JetpackBenefits = props => {
+	const { siteBenefits } = props;
+
+	return (
+		<React.Fragment>
+			{ siteBenefits.length > 0 && (
+				<React.Fragment>
+					<div className="jp-connection__disconnect-dialog__step-copy">
+						<p className="jp-connection__disconnect-dialog__large-text">
+							{ __(
+								'Jetpack is currently powering features on your site. Once you disconnect Jetpack, these features will no longer be available and your site may no longer function the same way.',
+								'jetpack'
+							) }
+						</p>
+					</div>
+					<div className="jp-connection__disconnect-card__group">
+						{ siteBenefits.map( ( { value, description, title } ) => (
+							<DisconnectCard title={ title } value={ value } description={ description } />
+						) ) }
+					</div>
+				</React.Fragment>
+			) }
+			{ siteBenefits.length <= 2 && (
+				<div className="jetpack-benefits__general-benefits-section">
+					<p className="jp-connection__disconnect-dialog__large-text">
+						{ __(
+							'Jetpack has many powerful tools that can help you achieve your goals',
+							'jetpack'
+						) }
+					</p>
+					<ul className="jetpack-benefits__general-benefits-list">
+						<li key="reason-cdn">
+							{ createInterpolateElement(
+								__(
+									'Speed up your site and provide mobile-ready images with <a>our CDN</a>',
+									'jetpack'
+								),
+								{
+									a: (
+										<a
```

Comment:
I wonder if we should use `ExternalLink` for those links, so we get an icon clearly showing that the links are opening in a new window?

Reply:
TIL about `ExternalLink`! Sounds like a good idea, will swap these out.